README_JP6.md

```diff
@@ -160,6 +161,8 @@ sudo /opt/nvidia/jetson-io/config-by-hardware.py -n 2="Jetson RealSense Camera D
 
 # Enable d4xx autoload:
 echo "d4xx" | sudo tee /etc/modules-load.d/d4xx.conf
+# Enable depmod scan for "extra" modules
+sudo sed -i 's/search updates/search extra updates/g' /etc/depmod.d/ubuntu.conf
 # update driver cache
 sudo depmod
```

Comment:
@dmipx  Let's have Arun run this flow and validate it?
He should get the JP4 jetpack by now